crates/weaver_live_check/src/live_checker.rs

```diff
@@ -535,7 +535,7 @@ mod tests {
 
         let mut stats = LiveCheckStatistics::new(&live_checker.registry);
         for sample in &mut samples {
-            sample.run_live_check(&mut live_checker, &mut stats);
+            let _ = sample.run_live_check(&mut live_checker, &mut stats);
```

Comment:
Nit: in the future we should not ignore the failures here, even for tests.  Instead the test can return `Result`, and you can `?` your way out of it.

Reply:
Fixed in #727